docs/examples/Basic_Operations.ipynb

```diff
@@ -52,7 +52,10 @@
    "cell_type": "code",
    "execution_count": null,
    "metadata": {
-    "id": "3njsiY73m7_V"
+    "id": "3njsiY73m7_V",
+    "pycharm": {
```

Comment:
I'm not sure what the new pycharm blocks are.  It probably doesn't make a difference for the output or functionality when running in jupyterhub.

Reply:
I assume PyCharm somehow injected these blocks when @horsburgh was editing this file. I removed those blocks.